src/Symfony/Component/TypeInfo/Tests/Type/ObjectTypeTest.php

```diff
@@ -35,4 +35,12 @@ public function testIsIdentifiedBy()
 
         $this->assertTrue((new ObjectType(self::class))->isIdentifiedBy('array', 'object'));
     }
+
+    public function testAccepts()
+    {
+        $this->assertFalse((new ObjectType(self::class))->accepts('string'));
+        $this->assertFalse((new ObjectType(self::class))->accepts(new \stdClass()));
+        $this->assertTrue((new ObjectType(TestCase::class))->accepts($this));
```

Comment:
Using `parent::class` would be the same, but more explicit in the purpose of the assertion.
```suggestion
        $this->assertTrue((new ObjectType(parent::class))->accepts($this));
```